openwisp_notifications/tasks.py

```diff
@@ -25,3 +27,15 @@ def delete_obsolete_notifications(instance_app_label, instance_model, instance_i
         | Q(target_object_id=instance_id)
     )
     Notification.objects.filter(where).delete()
+
+
+@shared_task
+def delete_old_notifications():
+    """
+    Delete notifications having 'timestamp' more than 90 days scheduled every day.
+    """
+    where = (
+        Q(timestamp<=timezone.now() - timedelta(days=90))
```

Comment:
timestamp__lte=timezone.now()

Reply:
Will this `timstamp<= timezone.now() - days` do?

Reply:
no, please follow the instructions we left in the last 2 reviews